models/importCsvData.js

```diff
@@ -0,0 +1,265 @@
+// we can also copy & paste excel data for batch task
```

Comment:
Could you write an abstract comment about how this importer work generally speaking?